api/common/context.go

```diff
@@ -26,14 +26,19 @@ import (
 	"github.com/loadimpact/k6/core"
 )
 
+// ContextKey is another name for int.
 type ContextKey int
```

Comment:
```suggestion
type contextKey int
```
I think that, instead of commenting this with a useless comment, the type can safely be unexported. It's not going to be used anywhere else, after all, that's why we have the helper functions below

Reply:
Yes, thanks!